ihmc-avatar-interfaces/src/main/java/us/ihmc/avatar/wholeBodyHardwareControl/AvatarMultiThreadingFactory.java

```diff
@@ -233,6 +240,32 @@ public AvatarMultiThreadingManager buildThreadsAndThreadingManager()
       // Set up the block to prevent execution whenever there is no new state message.
       threadingManager.get().setBlockingProvider(() -> !hardwareCommunicationInterface.hasNewStateMessage());
 
+      if (logLocally)
+      {
+         // Setup logger
+         ArrayList<RegistrySendBufferBuilder> builders = new ArrayList<>();
+         builders.add(new RegistrySendBufferBuilder(rootRegistry, null));
+         builders.add(new RegistrySendBufferBuilder(controllerThread.get().getYoVariableRegistry(), null, controllerThread.get().getSCS2YoGraphics()));
+         if (stepGeneratorThread.hasValue())
+         {
+            builders.add(new RegistrySendBufferBuilder(stepGeneratorThread.get().getYoVariableRegistry(), null, stepGeneratorThread.get().getSCS2YoGraphics()));
+         }
+         if (ikStreamingThread.hasValue())
+         {
+            builders.add(new RegistrySendBufferBuilder(ikStreamingThread.get().getYoVariableRegistry(), null, ikStreamingThread.get().getSCS2YoGraphics()));
+         }
+
+         // Logging locally on the robot
+         intraprocessYoVariableLogger = new IntraprocessYoVariableLogger(getClass().getSimpleName(), robotModel.getLogModelProvider(), builders, 100000, 0.01);
+         intraprocessYoVariableLogger.start();
+
+         threadingManager.get()
+                         .addPostEstimatorThreadRunnable(() -> intraprocessYoVariableLogger.update(estimatorThread.get()
+                                                                                                                  .getHumanoidRobotContextData()
+                                                                                                                  .getTimestamp()));
+         System.out.println("Logging Status: data is being logged LOCALLY");
```

Comment:
```suggestion
         LogTools.info("Logging locally.");
```
There's no need to shout :P

Reply:
to be fair, it would be easier to see

Reply:
Yeah but what about all the other messages? This feels like a debugging thing to all caps it